ci/docker/build/Dockerfile

```diff
@@ -2,24 +2,22 @@ FROM ubuntu:16.04
 
 ENV DEPOT_TOOLS_PATH $HOME/depot_tools
 ENV ENGINE_PATH $HOME/engine
-
-RUN apt-get update
-RUN apt-get install -y git wget curl unzip python lsb-release sudo apt-transport-https
-
-RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH
 ENV PATH $PATH:$DEPOT_TOOLS_PATH
 
-RUN mkdir --parents $ENGINE_PATH
+RUN apt-get update && \
```

Comment:
I would suggest you to combine it with RUN on line 31 if possible so you have a single RUN with apt update / install / clean

Reply:
There seems to be a chicken'n'egg problem in that this step installs `curl`, which is used below to configure the `apt` keyrings before we install more stuff :thinking: